### PR TITLE
test(torch_nntile): enable formerly skipped graph parity tests

### PR DESCRIPTION
**Pull request by Muxas**
## Summary

Re-enabled the graph-mode parity tests that were skipped after the staging I/O redesign. Most now pass without code changes beyond a few targeted fixes.

**Newly passing (24 tests):**
- `test_repeat_identity_matches_cpu`
- Softmax backward (3 tests)
- LayerNorm / RMSNorm backward (5 tests)
- Embedding backward (2 tests)
- Full GPT-2 minimal + stock HF graph tests (13 tests)
- `test_gpt2_lm_head_graph_mode_forward` (was xfail)

**Fixes included:**
- Use `torch.ones_like(y_nnt)` for constant grad outputs instead of CPU `.to("nntile")` ingress
- `norm_backward`: register param-grad aliases on the logical node
- Optimizer steps: shape-bridge grad graph nodes to param layout before SGD/Adam
- Embedding tests: `load_state_dict` on CPU before `.to("nntile")` (no double ingress)
- GPT-2 graph subprocess: reference HF on `input_ids.cpu()`

**Still skipped (4):**
- `test_context_apis_require_libnntile` — intentional (stub build only)
- `test_cross_entropy_ignore_index_matches_cpu` — needs graph-native valid-label count for mean scale (not `1/numel`)
- `test_global_norm_backward_*` (2) — gather abort when bridging 1D flat grad node to 2D param layout

## Test plan

- [x] Full `pytest` in `torch_nntile/`: **215 passed, 4 skipped**

<div><a href="https://cursor.com/agents/bc-019f21af-2c4d-7787-af73-2902822994e3"><picture><source media="(prefers-color-scheme: dark)" srcset="https://cursor.com/assets/images/open-in-web-dark.png"><source media="(prefers-color-scheme: light)" srcset="https://cursor.com/assets/images/open-in-web-light.png">Open in Web</picture></a>&nbsp;<a href="https://cursor.com/background-agent?bcId=bc-019f21af-2c4d-7787-af73-2902822994e3"><picture><source media="(prefers-color-scheme: dark)" srcset="https://cursor.com/assets/images/open-in-cursor-dark.png"><source media="(prefers-color-scheme: light)" srcset="https://cursor.com/assets/images/open-in-cursor-light.png">Open in Cursor</picture></a>&nbsp;</div>

### CODE REVIEW

**Comment by Muxas** [comment]
this PR was created by Cursor by mistake.
